client/src/app/layout/AppAboutModal/AppAboutModal.tsx

```diff
@@ -24,44 +25,46 @@ export const AppAboutModal: React.FC<AppAboutModalProps> = ({
   onClose,
 }) => {
   const { t } = useTranslation();
-
+  const brandName = APP_BRAND === BrandType.Konveyor ? "Tackle" : "MTA";
   return (
     <AboutModal
       isOpen={isOpen}
       onClose={onClose}
       trademark="COPYRIGHT © 2022."
-      brandImageSrc={brandImage}
+      brandImageSrc={
+        APP_BRAND === BrandType.Konveyor ? tackleBrandImage : mtaBrandImage
+      }
       brandImageAlt="Logo"
-      productName="Tackle"
+      productName={brandName}
     >
       <TextContent className="pf-u-py-xl">
-        <Text component={TextVariants.p}>{t("about.introduction")}</Text>
-        <Text component={TextVariants.p}>{t("about.description")}</Text>
         <Text component={TextVariants.p}>
-          <Trans i18nKey="about.bottom1">
-            Tackle is a project within the{" "}
-            <Text
-              component={TextVariants.a}
-              href="https://www.konveyor.io/"
-              target="_blank"
-            >
-              Konveyor community
-            </Text>
-            .
-          </Trans>
+          {t("about.introduction", { brandType: brandName })}
         </Text>
         <Text component={TextVariants.p}>
-          <Trans i18nKey="about.bottom2">
-            For more information please refer to{" "}
-            <Text
-              component={TextVariants.a}
-              href="https://tackle-docs.konveyor.io/"
-              target="_blank"
-            >
-              Tackle documentation
-            </Text>
-            .
-          </Trans>
+          {t("about.description", { brandType: brandName })}
+        </Text>
+        <Text component={TextVariants.p}>
+          {t("about.bottom1", { brandType: brandName })}{" "}
+          <Text
+            component={TextVariants.a}
+            href="https://www.konveyor.io/"
+            target="_blank"
+          >
+            Konveyor community
+          </Text>
+          .
+        </Text>
+        <Text component={TextVariants.p}>
+          {t("about.bottom2")}{" "}
+          <Text
+            component={TextVariants.a}
+            href="https://tackle-docs.konveyor.io/"
```

Comment:
This will still link to the Tackle docs, do we have MTA docs we can link to or is this fine?

Reply:
Updates